docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -197,7 +202,7 @@
       block:
       - name: Find deployment-manager app helm-chart in application directory
         find:
-          paths: /usr/local/share/applications/helm/
+          paths: application_helm_chart_dir
```

Comment:
If defaults are not set, this task will fail due to not defined

Reply:
Handled